app/webpacker/packs/application.js

```diff
@@ -32,6 +32,21 @@ document.addEventListener("turbolinks:load", () => {
     $("#prayer_categories").selectize({
         maxItems: 10
     });
+
+    $('#prayer_status').selectize({
```

Comment:
@TimothyHou oh can you put all the prayers related javascripts in an external js file? Can get very messy if all in application.js